security/manager/chrome/pipnss/nsserrors.properties

```diff
@@ -133,7 +133,7 @@ SSL_ERROR_NEXT_PROTOCOL_NO_CALLBACK=Rozszerzenie negocjacji następnej wersji pr
 SSL_ERROR_NEXT_PROTOCOL_NO_PROTOCOL=Serwer nie obsługuje żadnego z protokołów ogłaszanych przez klienta w rozszerzeniu ALPN.
 SSL_ERROR_INAPPROPRIATE_FALLBACK_ALERT=Serwer odrzucił żądanie, ponieważ klient zmienił wersję TLS na niższą niż obsługiwana przez serwer.
 SSL_ERROR_WEAK_SERVER_CERT_KEY=Certyfikat serwera zawiera klucz publiczny o zbyt słabym szyfrowaniu.
-SEC_ERROR_IO=Podczas autoryzacji bezpieczeństwa wystąpił błąd wejścia/wyjścia.
+SEC_ERROR_IO=Podczas upoważniania bezpieczeństwa wystąpił błąd wejścia/wyjścia.
```

Comment:
This one is incorrect but I'm not sure how it should be fixed - one option would be to remove "bezpieczeństwa" maybe.